recipes/r-platetools/LICENSE.txt

```diff
@@ -0,0 +1,3 @@
+YEAR: 2016
+COPYRIGHT HOLDER: Scott Warchal
```

Comment:
That file looks strange. Its not the MIT license thats for sure :)

Reply:
yeah, good point. Maybe @Swarchal can clarify?

Reply:
It's unfortunately how CRAN want things formatted for submission. I can add an additional LICENSE.md file containing the full MIT license to the repo if that would fix things?

Reply:
In the meantime, I added the standard MIT license in 57db5c1

Reply:
Thanks @Swarchal! I must have written my previous response at the same time. Does the change in 57db5c1 look ok?

Reply:
If it doesn't need to pass CRAN checks then yes that's fine with me!